### PR TITLE
Lattice 1726

### PR DESCRIPTION
**Pull request by jokedurnez**
We got a basic service that has a scheduled job that can read the database.

At this point there's probably way too many beans in the pods. Planning on cleaning that up, when it's clearer which ones we'll use.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+ext.projectName = "transporter"
+ext.scmUrl = 'scm:git:https://github.com/openlattice/transporter.git'
+ext.connectionUrl = 'scm:git:https://github.com/openlattice/transporter.git'
+ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/transporter.git'
+
+//apply from: "https://raw.githubusercontent.com/openlattice/gradles/master/openlattice.gradle"
+
+/////////// ANYTHING BELOW SHOULD BE UNCOMMENTED AS SOON AS GIT STUFF ABOVE WORKS /////////
```
Comment (geekbeast) [inline build.gradle]: ?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```
Comment (geekbeast) [inline gradle/wrapper/gradle-wrapper.properties]: Should switch to 4.10.3

File: src/main/resources/rhizome.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+hazelcast:
+    server: false
+    group: "local"
+    password: "trellis"
+    seed-nodes: 
+        - "localhost"
+
+hikari:
+    jdbcUrl: jdbc:postgresql://localhost:5432/openlattice
+    username: "jokedurnez"
```
Comment (geekbeast) [inline src/main/resources/rhizome.yaml]: Use oltest so it works on other people's machines without configuration.

**Review by geekbeast — CHANGES_REQUESTED** [review]

**Comment by jokedurnez** [comment]
@geekbeast solved your comments.

**Pull request closed without merging**